.gitignore

```diff
@@ -14,7 +14,6 @@ gen/
 
 # Gradle files
 .gradle/
-gradle/
```

Comment:
The plugin I'm using, put that.
But the wrapper doesn't depends by the local library? isn't auto-generated then?